documentation/puppet-api/v4/compile.markdown

```diff
@@ -0,0 +1,94 @@
+---
+layout: default
+title: "Puppet Server: Puppet API: Compile"
+canonical: "/puppetserver/latest/puppet-api/v4/compile.html"
+---
+
+[PCore Generic Data]: https://github.com/puppetlabs/puppet-specifications/blob/a87c9967ebd7071e0a548920f3262ac24cc3f4cd/language/data-types/pcore-generic-data.md
+[`auth.conf` documentation]: ../../config_file_auth.markdown
+
+The compile endpoint allows compilation of arbitrary pcore serialized Puppet
+Code ASTs (aka parsed, but unevaluated Puppet Code snippets). In addition to
+the code AST, the caller must provide the certname, environment, facts,
+trusted facts, and variables for compilation (the context in which this AST
+will be evaluated). Optionally, the caller may provide job_id, transaction_id,
```

Comment:
Can you put the fields in this block in `backticks`? I think they should stand out a bit more.